.editorconfig

```diff
@@ -0,0 +1,17 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+# Change these settings to your own preference
+indent_style = space
+indent_size = 2
```

Comment:
I'm not sure that indent_size = 2 is correct in our case.

Reply:
It may be configured per file type. 
I've got an error with idents in json-file and thought that such ident is valid for all file types.